Realm.PCL/RealmPCL.cs

```diff
@@ -326,6 +326,62 @@ public RealmResults<dynamic> All(string className)
             return null;
         }
 
+
+        /// <summary>
+        /// Fast lookup of an object from a class which has a PrimaryKey property.
+        /// </summary>
+        /// <typeparam name="T">The Type T must not only be a RealmObject but also have been processd by the Fody weaver, so it has persistent properties.</typeparam>
```

Comment:
Typo: processd. But I would be tempted to just remove the note about Fody as I think it's a given -- you can't do any of the other operations with non-woven classes either.